.scalafmt.conf

```diff
@@ -26,3 +26,8 @@ fileOverride {
 project.excludeFilters = [
   "scalafix/*"
 ]
+rewrite.scala3.convertToNewSyntax = true
+runner.dialectOverride.allowSignificantIndentation = false
+runner.dialectOverride.allowAsForImportRename = false
+runner.dialectOverride.allowStarWildcardImport = false
+runner.dialectOverride.allowPostfixStarVarargSplices = false
```

Comment:
What if we set `rewrite.scala3.convertToNewSyntax` to `false` and just enable `allowQuestionMarkAsTypeWildcard`, will it work the same?

Reply:
Not same. 

>  we set rewrite.scala3.convertToNewSyntax to false and just enable allowQuestionMarkAsTypeWildcard

It's allow both syntax.